activation/post.go

```diff
@@ -390,7 +390,8 @@ func (mgr *PostSetupManager) commitmentAtx(ctx context.Context, dataDir string,
 		}
 
 		// if this node has not published an ATX select the best ATX with `findCommitmentAtx`
-		return mgr.findCommitmentAtx(ctx)
+		// TODO: replace with highestATX api call
```

Comment:
Just a note for later: the node-service already has an endpoint to query a positioning ATX. This can be used for the commitment ATX (it would be the same ATX).